customtransforms/custom_single_json_retriever.py

```diff
@@ -0,0 +1,29 @@
+from transformer import Transformer
+from utilities import flat
+
+
+class SingleJsonRetriever(Transformer):
+    '''
+    Transforms perfherder data into the standardized data format.
+    '''
+    entry_number = 0
+
+    def transform(self, data):
+        ret = []
+        self.entry_number += 1
+
+        flatted_data = flat(data, [])
+
+        for k, v in flatted_data.items():
+            data_obj = []
+            for i in v:
+                data_obj.append({'value': i, 'xaxis': self.entry_number})

```

Comment:
rewrite as a single expression:

```
            data_obj = [
                {'value': i, 'xaxis': self.entry_number}
                for i in v
            ]
```

also, the `xaxis` is not descriptive.  Maybe call it `entry_number`?